src/index.html

```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.2/dist/css/bootstrap.min.css" integrity="sha384-uWxY/CJNBR+1zjPWmfnSnVxwRheevXITnMqoEIeG1LJrdI0GlVs/9cVSyPYXdcSF" crossorigin="anonymous">
+  <script async src="https://www.googletagmanager.com/gtag/js?id=G-N99JP6TWYS"></script>
+  <script>
+  </script>
+  <title>Game Leaderboard</title>
+</head>
+<body>
+  <header><h1>Leaderboard</h1></header>
+<section class="scores d-flex">
+  <div id="scores" class="score">
+    <h2>Recent Score</h2>
+    <button type="submit">Refresh</button>
+  </div>
+  <div class="add-new" > 
+    <fieldset class="field">
+      <legend>Add your Score</legend>
+      <input type="text" id="name">
+      <input type="text" id="score">
+      <button type="submit">Submit</button>
+    </fieldset>
+  </div>
+</section>
+</body>
+</html>
```

Comment:
- [ ] Please update the code so it will look more similar to the original
Yours
![Screenshot from 2021-10-12 23-24-26](https://user-images.githubusercontent.com/62705947/137031300-8f6f417e-b1c8-4a93-8c31-4af805d12210.png)
Original:
![leader](https://user-images.githubusercontent.com/62705947/137031338-64235ee4-9f64-4bf4-b8b7-b9a2301b9d7a.png)

Reply:
@GzimAsani  Hi Man,

thank you for you comments, with the requirements they said **_(Write plain HTML markup with minimum styling (just to make the layout), because you will implement styles in the following steps._**
) 
So I don't think this problem is an **optional** not **blocking** issue for my pull request I'm right ?

Reply:
Hi @iammouaz,

Yeah I the comment is optional, just, please add a description in your pull request and you should be fine